.github/workflows/build.yml

```diff
@@ -313,5 +313,43 @@ jobs:
             docker.io/hashicorppreview/${{ env.PKG_NAME }}:${{ env.version }}-dev
             docker.io/hashicorppreview/${{ env.PKG_NAME }}:${{ env.version }}-${{ github.sha }}
 
+  minimum-os:
+    name: OS Compatibility
+    # A quick smoke test of our binaries on our minimum target OS (RHEL 7). Why RHEL 7? Because the glibc version is that old (2.17).
+    needs:
+      - get-go-version
+      - get-product-version
+      - build-linux
+    runs-on: ubuntu-latest
```

Comment:
Generally we prefer specific OS's; we've a let a lot of these creep in so I created https://github.com/hashicorp/nomad/issues/17714 to fixup them all.

Reply:
Good catch!